portal/build.sbt

```diff
@@ -0,0 +1,5 @@
+enablePlugins(JavaServerAppPackaging)
+mappings in Universal ++= Seq(
+  (baseDirectory in Compile).value / "src/main/resources/application.conf" -> "conf/application.conf",
+  (baseDirectory in Compile).value / "src/main/resources/logback.xml" -> "conf/logback.xml"
+)
```

Comment:
I think we don't have to have two `build.sbt` files in our project. You should configure native packager in the existing `build.sbt`.

Reply:
I don’t know for sure, but I think @ilyaonishenko can describe the reason

Reply:
I believe it's the style thing. 
If we're going to to continue with single set style, we can keep these changes in the global `build.sbt`

@Dmitrii-Shevchenko you can move it to the main `build.sbt`, but apply this changes **specifically** to the `portal` project.
It will look like 
```
lazy val portal = project
   ...
  .enablePlugins(JavaAppPackaging)
  ...
  .settings(
    ...
    mappings..
  )
```

Reply:
Done